include/effectmanager.h

```diff
@@ -310,19 +324,18 @@ class EffectManager : public IJSONSerializable
 
     void StartEffect()
     {
+        // If there's a temporary effect override from the remote control active, we start that, else
+        // we start the current regular effect
+
+        LEDStripEffect *pEffect = _ptrRemoteEffect ? _ptrRemoteEffect.get() : _vEffects[_iCurrentEffect];

```

Comment:
Perhaps this can become shared_ptr after merged so that it holds a reference to the remote effect.  Also, we should really name remoteEffect to "tempEffect" or similar.  I don't think it's naming is very clear that its from the temp remote control effect!

Reply:
Changing the naming of the effect did cross my mind, but as I didn't - and don't - know the history of the special one effect, I chose not to touch it. If you also think it should be changed, then I think we can take care of that when we implement the switch to shared_ptr as well.